darts/tests/test_probabilistic_models.py

```diff
@@ -68,27 +67,61 @@ def test_probabilistic_forecast_accuracy(self):
                                                                  self.constant_multivar_ts,
                                                                  self.constant_noisy_multivar_ts)
 
+    def test_probabilistic_forecast_risk(self):
+        for model_cls, model_kwargs, err in models_cls_kwargs_errs:
+            self.helper_test_probabilistic_forecast_risk(model_cls, model_kwargs, err,
+                                                             self.constant_ts, self.constant_noisy_ts)
+            if issubclass(model_cls, GlobalForecastingModel):
+                self.helper_test_probabilistic_forecast_risk(model_cls, model_kwargs, err,
+                                                                 self.constant_multivar_ts,
+                                                                 self.constant_noisy_multivar_ts)
+
     def helper_test_probabilistic_forecast_accuracy(self, model_cls, model_kwargs, err, ts, noisy_ts):
         model = model_cls(**model_kwargs)
         model.fit(noisy_ts[:100])
         pred = model.predict(n=100, num_samples=100)
 
-        # test accuracy of the median prediction compared to the noiseless ts
+        # test accuracy of the median prediction and quantile (=0.5) rho-risk compared to the noiseless ts
         mae_err_median = mae(ts[100:], pred)
         self.assertLess(mae_err_median, err)
 
-        # test accuracy for increasing quantiles between 0.7 and 1 (it should decrease)
+        # test accuracy for increasing quantiles between 0.7 and 1 (it should decrease, mae should increase),
         tested_quantiles = [0.7, 0.8, 0.9, 0.99]
         mae_err = mae_err_median
         for quantile in tested_quantiles:
             new_mae = mae(ts[100:], pred.quantile_timeseries(quantile=quantile))
             self.assertLess(mae_err, new_mae)
             mae_err = new_mae
 
-        # test accuracy for decreasing quantiles between 0.3 and 0 (it should decrease)
+        # test accuracy for decreasing quantiles between 0.3 and 0 (it should decrease, mae should increase),
         tested_quantiles = [0.3, 0.2, 0.1, 0.01]
         mae_err = mae_err_median
         for quantile in tested_quantiles:
             new_mae = mae(ts[100:], pred.quantile_timeseries(quantile=quantile))
             self.assertLess(mae_err, new_mae)
             mae_err = new_mae
+
+    def helper_test_probabilistic_forecast_risk(self, model_cls, model_kwargs, err, ts, noisy_ts):
+        model = model_cls(**model_kwargs)
+        model.fit(noisy_ts[:100])
+        pred = model.predict(n=100, num_samples=100)
+
+        # test rho-risk (at quantile=0.5) compared to the noiseless ts
+        rrisk_median = rho_risk(ts[100:], pred, rho=0.5)
+        self.assertLess(rrisk_median, err)
+
+        # test risk for increasing quantiles between 0.75 and 1 should increase
+        tested_quantiles = [0.8, 0.9, 0.95, 0.99]  # some test cases actually have minimum risk around 0.75
```

Comment:
Why is it that the rho risk is increasing when rho is higher? Sorry if this is obvious, I just briefly looked at the definitions and it's not immediately clear to me.

Reply:
Let's say your predicted stochastic series at quantile/rho=0.5 represents the actual series, then the more you move away from 0.5, the bigger the loss and the higher the risk becomes.
So in these examples with **tested** models and quantiles, the risk increases with each rho in `tested_quantiles`.

If with some other model your predicted series is closest to the actual series at quantile=1., then this test would fail (and also accuracy test `test_probabilistic_forecast_accuracy()`).

I hope this answered your question, if not just let me know :)

Reply:
Actually, yesterday @pennfranc noticed that the definition of the rho_metric in the originally used paper had a mistake.
From now on we use the rho_metric defined in [this paper](https://papers.nips.cc/paper/2016/hash/03255088ed63354a54e0e5ed957e9008-Abstract.html).

I'll update the pull request comment.